test/save_resume_test.py

```diff
@@ -153,8 +153,9 @@ def do_test(filename, args, verbose=None, repeat_args=None, known_failure=False)
         errors += do_test(filename, '--loss_function logistic --link logistic')
         errors += do_test(filename, '--nn 2')
         errors += do_test(filename, '--binary')
-        errors += do_test(filename, '--ftrl', known_failure=True)
+        errors += do_test(filename, '--ftrl')
         errors += do_test(filename, '--pistol', known_failure=True)
```

Comment:
Are pistol and coin still in known failure?

Reply:
Yes. As I write above, the data structure is saved correctly, but the weights seem to be different if trained continuously or trained, saved, resumed. The difference does not happen immediately, but after some samples. This is why I suspect numerical issues.

Reply:
This violates my understanding of computers so I suspect there is something that we're missing.  I probably won't be able to debug before the next release, but it should get done...

Reply:
I am trying to debug this issue a bit more